contrib/infiniband/ibv_internal.h

```diff
@@ -1,10 +1,26 @@
 /*! \file ibv_internal.h */
 #include <infiniband/verbs.h>
 #include <stdbool.h>
+#include <inttypes.h>
 #include "ibvidentifier.h"
 #include "lib/list.h"
 #include "debug.h"
 
+/* Two 64-bit random numbers to verify whether
+ a struct is shadowed or real. */
```

Comment:
@jiajuncao, Here is a more verbose comment to replace the above lines.  I'd like to propose this to help other readers of the code, who don't know the background.
```
/* Two 64-bit fixed, random numbers to verify whether a struct is shadowed 
 * (struct internal_ibv_dev) or real (struct ibv_dev).  This signature
 * allows us to detect recursive cases:  A target application may call our
 * wrapper using the shadowed struct.  We then pass on the real struct to the
 * real function in the IB library.  But if the real function calls a second
 * real IB function, then it will pass the real struct to our wrapper 
 * for the second IB function. 
 */
```